rows/utils.py

```diff
@@ -297,3 +310,53 @@ def export_to_uri(table, uri, *args, **kwargs):
         raise ValueError('Plugin (export) "{}" not found'.format(plugin_name))
 
     return export_function(table, uri, *args, **kwargs)
+
+
+def decompress(path, **kwargs):
```

Comment:
Could you please add an `algorithm` parameter to this function? It should defaults to `None` (if `None`, use file extension to define it).